### ISSUE TITLE
Configure language installer

### ISSUE THREAD

**Issue by HumanG33k** [fvarrui/JavaPackager#48]
Maybe it's not related to JavaPackager but only languages present for installer (windows) :
- English 
- Spanish

**Comment by fvarrui** [comment]
Yes, it's related to JavaPackager 😄 .

There's a VTL template for Inno Setup [here](https://github.com/fvarrui/JavaPackager/blob/master/src/main/resources/windows/iss.vtl).

You can create your custom template in `${basedir}\assets\windows\iss.vtl` and add more languages in the `Languages` category:
```
[Languages]
Name: "english"; MessagesFile: "compiler:Default.isl"
Name: "french"; MessagesFile: "compiler:Languages\French.isl"
Name: "german"; MessagesFile: "compiler:Languages\German.isl"
...
```
I've got these available:

![image](https://user-images.githubusercontent.com/4354486/90313839-89547480-df07-11ea-999f-20f9ffd8b680.png)

**Comment by fvarrui** [comment]
Hi @HumanG33k!
I'm not sure if you could try this new feature, but it's now published in v1.2.0.
Thanks!
